### DISCUSSION TITLE
Integrate with oracle

### DISCUSSION THREAD

**Discussion by mateivladimir** [burningalchemist/sql_exporter#443] [Ideas]
Hi. I've tried to connect to an oracle database and i cant figure what i've missed. I've guessed it doesnt exist a driver for connecting to oracle databases yet. It is true? Does anyone try to do that ?

**Comment by burningalchemist** [comment]
Hey @mateivladimir, it works well with Oracle, but you need to add a driver yourself to `drivers.go`, and build the binary. For Oracle DB, you might want to use a driver from the list in the reference (e.g. https://github.com/sijms/go-ora). 

Have a look at the documentation, and feel free to reach out if any help is needed. :)

Reference for supported drivers can be found here: https://github.com/xo/dburl
